oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/AbstractLog.java

```diff
@@ -18,25 +18,29 @@
 
 package org.apache.skywalking.oap.server.core.source;
 
+import java.util.ArrayList;
+import java.util.List;
 import lombok.Getter;
 import lombok.Setter;
 import org.apache.skywalking.oap.server.core.UnexpectedException;
+import org.apache.skywalking.oap.server.core.analysis.manual.searchtag.Tag;
 import org.apache.skywalking.oap.server.core.query.type.ContentType;
 
 @Setter
 @Getter
 public abstract class AbstractLog extends Source {
     private long timeBucket;
```

Comment:
`abstract class Source` includes this already.

Reply:
done